app/components/candidate_interface/other_qualifications_review_component.rb

```diff
@@ -31,6 +31,14 @@ def other_qualifications_rows(qualification)
       end
     end
 
+    def no_qualification_row
+      [{
```

Comment:
Does this need to be in an array?

Reply:
Yeah I think so, otherwise this happens to the layout:
![image](https://user-images.githubusercontent.com/47917431/114598759-2a0f5580-9c8a-11eb-8578-71dcb30e9184.png)